core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -368,8 +356,7 @@ private[spark] object JsonProtocol {
     ("Shuffle Read Metrics" -> shuffleReadMetrics) ~
     ("Shuffle Write Metrics" -> shuffleWriteMetrics) ~
     ("Input Metrics" -> inputMetrics) ~
-    ("Output Metrics" -> outputMetrics) ~
-    ("Updated Blocks" -> updatedBlocks)
```

Comment:
Good point, I'll look through some older versions and test out a few things with this.